### ISSUE TITLE
Null values passed to className prop will render the string "null"

### ISSUE THREAD

**Issue by blord-fullscreen** [glennflanagan/react-collapsible#71]
Code like this:
``` 
       const specialClass = somevar ? 'special-style' : null;

        return (
            <Collapsible
                classParentString="dbc-split-nav-item"
                className={specialClass}
            >
               Hello there.
            </Collapsible>
       );
```

winds up applying the class `"null"` when `someVar` is false. This is not idiomatic in React, where `null` values are generally ignored. I discovered this by accident when browsing the inspector. No harm done, but very strange :) One workaround is to set the class to an empty string, but that's a bit weird in this context.

Thanks for a helpful package.

**Comment by glennflanagan** [comment]
As you said it, you can pass an empty string as a workaround.

The class names in the collapsible are being generated for all the different states. Feel free to open a PR for the enhancement.

**Comment by karltaylor** [comment]
I'll try take a look at this and get it merged into v2.1.0

**Comment by karltaylor** [comment]
Closing this as passing an empty string will suffice.
